composer/trainer/trainer_hparams.py

```diff
@@ -231,6 +228,14 @@ def validate(self):
                 f"Eval batch size ({self.eval_batch_size}) not divisible by the total number of processes ({world_size})."
             )
 
+        # There should be at least one structure for validation - evaluators or ordinary validation dataset
+        if self.evaluators is None or len(self.evaluators) == 0:
+            if self.val_dataset is None:
+                raise ValueError("val_dataset and evaluators shouldn't both be empty")
```

Comment:
We have a warning for this in the trainer. Can you remove this check here? Please keep the one below though.